src/Components/Navbar/Style1/index.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React from './node_modules/react';
 import NavbarContainer from '../../../Atoms/Navbar'
-import styled from 'styled-components';
+import styled from './node_modules/styled-components';
```

Comment:
Remove `./node_modules/` since a folder called `node_modules` does not exist in the Style1 folder. Or use full path to root `node_modules`.